modules/connectivity/ParticipantConnectionStatus.js

```diff
@@ -363,7 +363,7 @@ export default class ParticipantConnectionStatus {
             `Figure out conn status, is video muted: ${isVideoMuted
                  } is active(jvb): ${isConnActiveByJvb
                  } video track frozen: ${isVideoTrackFrozen
-                 } is in last N: ${isInLastN}
+                 } is in last N: ${isInLastN
```

Comment:
does this carrying over the "}" sign get rid of the newline in the printed message ?